pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -731,7 +736,7 @@ func configureVifV6Addresses(b *MasqueradeBindMechanism, err error) error {
 }
 
 func (b *MasqueradeBindMechanism) startDynamicIPServers(vmi *v1.VirtualMachineInstance) error {
-	if b.vif.IPv6.IPNet != nil {
+	if b.vif.IPv6.IPNet != nil && b.vif.DynamicIPv6 {
```

Comment:
Dhcpv6 is still started, so I'm not sure `DynamicIPv6` is a good name.

Reply:
Sounds good; looking at https://github.com/kubevirt/kubevirt/blob/606ee025ee8c80018678063aa66da586222a8bfa/pkg/virt-config/feature-gates.go#L26 I would name the key `RAGate` and the value `RouterAdvertisementSupport` . Your thoughts ?

Reply:
Is it better now ?